Sources/SwiftUINavigation/Documentation.docc/Articles/DestructuringViews.md

```diff
@@ -24,19 +24,19 @@ struct EditView: View {
 
   var body: some View {
     Form {
-      IfLet(self.$editableString) { $string in
-        TextField("Edit string", text: $string)
+      IfLet(self.$editableString) { $tempString in
+        TextField("Edit string", text: $tempString)
```

Comment:
A state variable named `string` already exists.
Just to clarify, I renamed this temporary variable `$string` → `$tempString`

Reply:
While I think we're totally fine shadowing that variable here, I think we're also down to avoid it. Instead of `$tempString`, maybe we could use something more like `$text` throughout these examples?

Reply:
Maybe we'd better avoid names including `string` to a String type variables.
How about this?

```suggestion
      IfLet(self.$editableText) { $text in
        TextField("Edit text", text: $text)
```

or, we can use `$tempText` or `$editedText`, etc. What do you think?